suricata.yaml.in

```diff
@@ -952,7 +952,15 @@ app-layer:
       #enabled: yes
     ssh:
       enabled: yes
-      #hassh: yes
+      # hassh: no
+
+      # What to do when the encrypted communications start:
+      # - default: keep tracking but stop inspection
```

Comment:
Should not be named `default` but how it behaves like `stop_inspect_only`, right ?

Reply:
Well, I wanted to use the same naming as with TLS. There it is the same. Maybe I could change it there as well so the default is still supported but obsoleted.

Reply:
And replaced with "track-only"